<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar</class>
 <widget class="QWidget" name="Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="calendarWidget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>192</y>
     <width>833</width>
     <height>475</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 243, 255)</string>
   </property>
   <widget class="QWidget" name="noteWidget" native="true">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>48</y>
      <width>261</width>
      <height>425</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>8</y>
       <width>253</width>
       <height>413</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="noteMenu" stretch="1,15">
      <item>
       <widget class="QWidget" name="noteWidgetMenu" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="noteList" native="true">
        <widget class="QListWidget" name="noteListWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>251</width>
           <height>380</height>
          </rect>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="calendarFormWidget" native="true">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>48</y>
      <width>551</width>
      <height>425</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>8</y>
       <width>541</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="calendarForm" rowstretch="10,10,10,10,10,10,10" columnstretch="10,10,10,10,10,10,10,10">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="CalendarCell" name="L0_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="CalendarCell" name="L2_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="CalendarCell" name="L7_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="CalendarCell" name="L3_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="CalendarCell" name="L2_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CalendarCell" name="L0_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="7">
       <widget class="CalendarCell" name="L7_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="CalendarCell" name="L3_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="CalendarCell" name="L4_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="CalendarCell" name="L2_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="CalendarCell" name="L3_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="CalendarCell" name="L3_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="CalendarCell" name="L7_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="CalendarCell" name="L0_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="CalendarCell" name="L6_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="CalendarCell" name="L6_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="CalendarCell" name="L0_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CalendarCell" name="L0_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="CalendarCell" name="L6_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="CalendarCell" name="L6_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="CalendarCell" name="L4_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="CalendarCell" name="L6_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="CalendarCell" name="L2_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="CalendarCell" name="L7_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="CalendarCell" name="L5_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="CalendarCell" name="L5_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="CalendarCell" name="L0_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="CalendarCell" name="L6_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="CalendarCell" name="L5_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="CalendarCell" name="L0_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CalendarCell" name="L1_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CalendarCell" name="L1_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CalendarCell" name="L1_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="CalendarCell" name="L1_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="CalendarCell" name="L7_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="CalendarCell" name="L4_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="CalendarCell" name="L5_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="CalendarCell" name="L7_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="CalendarCell" name="L1_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="CalendarCell" name="L2_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="CalendarCell" name="L3_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="CalendarCell" name="L4_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="CalendarCell" name="L5_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="CalendarCell" name="L1_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="CalendarCell" name="L2_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="CalendarCell" name="L3_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="CalendarCell" name="L4_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="CalendarCell" name="L5_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="CalendarCell" name="L1_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="CalendarCell" name="L2_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="CalendarCell" name="L3_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="CalendarCell" name="L4_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="CalendarCell" name="L5_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="CalendarCell" name="L6_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="7">
       <widget class="CalendarCell" name="L7_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="CalendarCell" name="L4_0">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="Datelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekLabel">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>0</y>
      <width>253</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
   </widget>
   <widget class="QWidget" name="menuBar" native="true">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>0</y>
      <width>149</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>68</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(236, 255, 226);
border-color: rgb(240, 220, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>40</y>
     <width>251</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalendarCell</class>
   <extends>QLabel</extends>
   <header location="global">toolCell.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
